### ISSUE TITLE
Extend support not only for DefaultCache behavior

### ISSUE THREAD

**Issue by stefanosbou** [benmarch/update-lambda-edge#3]
Are there any immediate plans to support other CacheBehaviors and not only DefaultCacheBehavior?

**Comment by benmarch** [comment]
I've pretty much just been waiting for someone to ask because I've only used the default cache behavior 😄 

Happy to add the feature, but could I ask you for an example of how you've configured a CF distro with an additional cache behavior? Ideally, if you could call the `cloudfront.getDistributionConfig()` API (via SDK or REST) and send me the data structure (and redact any confidential info) I can make the change pretty quickly. Alternatively, if you could instruct me on the best way to use an additional cache behavior, I can set it up myself. Thanks!

**Comment by stefanosbou** [comment]
Opened a PR https://github.com/benmarch/update-lambda-edge/pull/4

This is the output for getDistributionConfig
```
{
  "ETag": "XXXXXXXXXXX",
  "DistributionConfig": {
    "CallerReference": "XXXXXXXXXX",
    "Aliases": {
      "Quantity": 1,
      "Items": [
        "cdn.XXX.com"
      ]
    },
    "DefaultRootObject": "XXX.png",
    "Origins": {
      "Quantity": 1,
      "Items": [
        {
          "Id": "XXX",
          "DomainName": "XXX.s3.eu-west-1.amazonaws.com",
          "OriginPath": "",
          "S3OriginConfig": {
            "OriginAccessIdentity": "origin-access-identity/cloudfront/XXX"
          },
          "ConnectionAttempts": 3,
          "ConnectionTimeout": 10,
          "OriginShield": {
            "Enabled": false
          }
        }
      ]
    },
    "OriginGroups": {
      "Quantity": 0,
      "Items": [
        
      ]
    },
    "DefaultCacheBehavior": {
      "TargetOriginId": "XXX",
      "TrustedSigners": {
        "Enabled": false,
        "Quantity": 0,
        "Items": [
          
        ]
      },
      "TrustedKeyGroups": {
        "Enabled": false,
        "Quantity": 0,
        "Items": [
          
        ]
      },
      "ViewerProtocolPolicy": "redirect-to-https",
      "AllowedMethods": {
        "Quantity": 3,
        "Items": [
          "HEAD",
          "GET",
          "OPTIONS"
        ],
        "CachedMethods": {
          "Quantity": 2,
          "Items": [
            "HEAD",
            "GET"
          ]
        }
      },
      "SmoothStreaming": false,
      "Compress": true,
      "LambdaFunctionAssociations": {
        "Quantity": 0,
        "Items": [
          
        ]
      },
      "FieldLevelEncryptionId": "",
      "CachePolicyId": "XXX"
    },
    "CacheBehaviors": {
      "Quantity": 1,
      "Items": [
        {
          "PathPattern": "/*",
          "TargetOriginId": "XXX",
          "TrustedSigners": {
            "Enabled": false,
            "Quantity": 0,
            "Items": [
              
            ]
          },
          "TrustedKeyGroups": {
            "Enabled": false,
            "Quantity": 0,
            "Items": [
              
            ]
          },
          "ViewerProtocolPolicy": "redirect-to-https",
          "AllowedMethods": {
            "Quantity": 3,
            "Items": [
              "HEAD",
              "GET",
              "OPTIONS"
            ],
            "CachedMethods": {
              "Quantity": 2,
              "Items": [
                "HEAD",
                "GET"
              ]
            }
          },
          "SmoothStreaming": false,
          "Compress": false,
          "LambdaFunctionAssociations": {
            "Quantity": 2,
            "Items": [
              {
                "LambdaFunctionARN": "arn:aws:lambda:us-east-1:XXX:function:XXX:1",
                "EventType": "origin-response",
                "IncludeBody": false
              },
              {
                "LambdaFunctionARN": "arn:aws:lambda:us-east-1:XXX:function:XXX:1",
                "EventType": "viewer-request",
                "IncludeBody": false
              }
            ]
          },
          "FieldLevelEncryptionId": "",
          "CachePolicyId": "XXX"
        }
      ]
    },
    "CustomErrorResponses": {
      "Quantity": 0,
      "Items": [
        
      ]
    },
    "Comment": "XXX",
    "Logging": {
      "Enabled": false,
      "IncludeCookies": false,
      "Bucket": "",
      "Prefix": ""
    },
    "Enabled": true,
    "Restrictions": {
      "GeoRestriction": {
        "RestrictionType": "none",
        "Quantity": 0,
        "Items": [
          
        ]
      }
    },
    "WebACLId": "",
    "HttpVersion": "http2",
    "IsIPV6Enabled": true
  }
}
```

**Comment by benmarch** [comment]
Released v1.1.0 to support this feature. @stefanosbou please update and verify functionality.

**Comment by stefanosbou** [comment]
It works! Thanks a lot! 💯
